src/Admin/AbstractAdmin.php

```diff
@@ -14,9 +14,8 @@
 namespace Sonata\AdminBundle\Admin;
 
 use Doctrine\Common\Util\ClassUtils;
-use Knp\Menu\FactoryInterface as MenuFactoryInterface;
+use Knp\Menu\FactoryInterface;
 use Knp\Menu\ItemInterface;
-use Knp\Menu\ItemInterface as MenuItemInterface;
```

Comment:
Looks like there is an occurrence of `MenuItemInterface` somewhere in that file, please change it to `ItemInterface`

Reply:
My bad. Thank you!